app/controllers/concerns/vet360/writeable.rb

```diff
@@ -67,5 +67,14 @@ def render_new_transaction!(type, response)
 
       render json: transaction, serializer: AsyncTransaction::BaseSerializer
     end
+
+    def request_is_delete?
```

Comment:
Did you decide against the `Vet360::Deleteable`?

Reply:
I'm not opposed to it but it didn't seem necessary to me... After studying the existing concern, this added method seemed to fit in. So I didn't see any value in splitting off a new concern. Do you have strong feelings about this?

Reply:
I lean towards creating the new concern.  @rlbaker ?